dojo/models.py

```diff
@@ -2062,6 +2062,11 @@ class Finding(models.Model):
                                verbose_name=_('Service'),
                                help_text=_('A service is a self-contained piece of functionality within a Product. This is an optional field which is used in deduplication of findings when set.'))
 
+    planned_remediation_date = models.DateField(null=True,
+                                                editable=True,
+                                                verbose_name=_('Planned Remediation Date'),
+                                                help_text=_("The date the flaw is expected to be remediated."))
```

Comment:
```suggestion
                                                help_text=_("The date the finding is expected to be remediated."))
```

We never use the word flaw, so it should be finding to be consistent